### PR TITLE
NEPT-619: Add the Pull Request template.

### PR DESCRIPTION
**Pull request by voidtek**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Comment by imanoleguskiza** [comment]
👍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## NEPT-[Insert the number ticket here]
```
Comment (jfhovinne) [inline .github/PULL_REQUEST_TEMPLATE.md]: "Insert ticket number here" instead of "Insert the number ticket here"

**Comment by jfhovinne** [comment]
:+1:

**Comment by drupol** [comment]
A small nit pick, shouldn't it be "changelog" instead of "change log" ?

**Comment by jfhovinne** [comment]
@drupol See http://keepachangelog.com/

**Comment by jfhovinne** [comment]
Should be "Change log" but not a blocker :)

**Comment by jfhovinne** [comment]
:+1:

**Comment by drupol** [comment]
It's just weird to see it written in two words... first time I see it.
But no worries, it will not keep me from sleeping if it stays that way.

**Comment by jfhovinne** [comment]
:+1:

**Comment by drupol** [comment]
👍

**Pull request merged**